docs/setup/independent/install-kubeadm.md

```diff
@@ -164,6 +164,22 @@ apt-get install -y kubelet kubeadm kubectl
 
 {% endcapture %}
 
+{% capture debian %}
+
+```bash
+apt-get update && apt-get install -y apt-transport-https
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+cat <<EOF >/etc/apt/sources.list.d/kubernetes.list
+deb http://apt.kubernetes.io/ kubernetes-stretch main
+EOF
+apt-get update
+apt-get install -y kubelet kubeadm kubectl
+echo "Only kubectl are found. 2017-10-20"
+```
```

Comment:
I don't understand the echo line. @luxas Can you chime in here.

Reply:
Hi, I think the echo should not remain. What I want to say is that for Debian 9 only the package kubectl are on the repository. So the line 176 should be
176+ apt-get install -y kubectl

I think the best option to explain is make a table on the beginning on the page

| Distributionen Suported | kubelet | kubeadm | kubectl | 
| Ubuntu 16.04                  |  ✓        | ✓             | ✓        |
| Debian 9                          | ✗        | ✗              | ✓        |
...

But that could be another merge. Anyway, thanks for the documentation. The tutorials are great.